icaruscode/CRT/CRTDecoder/DecoderICARUSCRT_module.cc

```diff
@@ -119,7 +127,7 @@ uint64_t crt::DecoderICARUSCRT::CalculateTimestamp(icarus::crt::BernCRTTranslato
 void crt::DecoderICARUSCRT::produce(art::Event& evt)
 {
 
-  // Implementation of required member function here.
+  // Implementation of required member function here.'
```

Comment:
Spurious apostrophe.
```suggestion
  // Implementation of required member function here.
```